liblouis/compileTranslationTable.c

```diff
@@ -1141,14 +1141,18 @@ parseChars(FileInfo *nested, CharsString *result, CharsString *token) {
 					ch = QUOTESUB;
 					break;
 				case 'X':
+				  compileWarning(nested, "\\Xhhhh (with a capital 'X') is deprecated.");
+				  break;
```

Comment:
I thought you wanted to keep it backwards compatible, so then the `break` shouldn't be there. For me it's fine either way.

Reply:
Ah, you're right. If I do that I might as well report an error. I'll remove the break instead.